app/components/buttons/outlined-button.jsx

```
@@ -27,7 +28,7 @@
         showIcon = false;
     }
     return(
-        <button className={`text-btn flex border-2 rounded-full border-[--md-sys-color-outline] items-center justify-stretch focus:ring ${props.className}`} role='button' onClick={props.onClick}>
+        <button className={`text-btn flex border-2 rounded-full border-[--md-sys-color-outline] items-center justify-stretch focus:ring ${props.className}`} role='button' onClick={props.onClick} type={props.type}>
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)